db/column_family.cc

```diff
@@ -995,6 +995,24 @@ Status ColumnFamilyData::SetOptions(
 }
 #endif  // ROCKSDB_LITE
 
+// REQUIRES: DB mutex held
+Env::WriteLifeTimeHint ColumnFamilyData::CalculateSSTWriteHint(int level) {
+  if (initial_cf_options_.compaction_style != kCompactionStyleLevel) {
+    return Env::WLTH_NOT_SET;
+  }
+  if (level == 0 || level == 1) {
+    return Env::WLTH_SHORT;
+  }
+  int base_level = current_->storage_info()->base_level();
+
+  // L1: short, L2: medium, ...
+  if (level - base_level >= 3) {
```

Comment:
Where do you set medium for level-2 ?

Reply:
that's the: 
level - base_level +
+                            static_cast<int>(Env::WLTH_SHORT)